polaris-core/src/main/java/org/apache/polaris/core/persistence/PolarisMetaStoreManager.java

```diff
@@ -376,4 +376,14 @@ ResolvedEntityResult refreshResolvedEntity(
       @Nonnull PolarisEntityType entityType,
       long entityCatalogId,
       long entityId);
+
+  /**
+   * Indicates whether this metastore manager implementation requires entities to be reloaded via
+   * {@link #loadEntitiesChangeTracking} in order to ensure the most recent versions are obtained.
+   *
+   * <p>Generally this flag is {@code true} when entity caching is used.
+   */
+  default boolean requiresEntityReload() {
+    return true;
+  }
```

Comment:
Based on the comment, I wonder if we could just use `cache == null`?

Reply:
That's what it looks like to me too, but I'd like to allow implementations to make this decision separately. We can revisit and remove this method when a NoSQL impl. is make available for review (if it happens to be redundat)... but so far it seems useful for our WIP code.